hail_search/test_search.py

```diff
@@ -388,10 +388,11 @@ async def test_both_sample_types_search(self):
             [VARIANT2_BOTH_SAMPLE_TYPES], sample_data=FAMILY_2_BOTH_SAMPLE_TYPE_SAMPLE_DATA_MISSING_PARENTAL_WGS,
             inheritance_mode=inheritance_mode, **COMP_HET_ALL_PASS_FILTERS, intervals=[variant2_interval]
         )
-        # Genome passes quality and inheritance exome fails inheritance (parental data shows variant is inherited).
+        # Genome passes quality and inheritance but exome fails inheritance (parental data shows variant is inherited).
+        # Variant is excluded from search results.
```

Comment:
I think this case is covered by an above test -

        # Variant1 in family_2 is de novo in exome but maternally inherited in genome.
        # Genome passes quality and inheritance, show genotypes for both sample types.

Reply:
The search criteria changes between these 2 test cases. We really need back-to-back tests that the exact same search returns different results when the paternal data is present/absent. Instead of running a dominant and recessive search each filtered to variant 1 and variant 2, it would probably be better to not use an interval filter at all and test that dominant and recessive search return the expected combo of variants with and without paternal data

Reply:
I did this^!